xgi/classes/hypergraph.py

```diff
@@ -513,20 +513,21 @@ def add_edges_from(self, ebunch_to_add, **attr):
 
         Parameters
         ----------
-        ebunch_to_add : iterable
+        ebunch_to_add : Iterable
 
             An iterable of edges, in one of the following formats.
 
-            * Format 1: Iterable of hashables (id1, id2, ...), or
+            * Format 1: Iterable (edge1_members, edge2_members), or
             * Format 2: 2-tuple (members, id), or
             * Format 3: 2-tuple (members, attr), or
             * Format 4: 3-tuple (members, id, attr),
+            * Format 5: a dict of the form {id: members}
```

Comment:
are format 2 and 5 not redundant?

Reply:
Yes they are! But I thought adding the dict would be nice since it is the format returned by `H.edges.members(dtype=dict)`. So you can easily add edges from another hypergrpah.